frontend/src/css/modules/main-header.scss

```
@@ -42,24 +42,24 @@
+    width: 100%;
     z-index: 1000;
     background-color: $dark-gray;
     color: $light-gray;
     text-align: center;
-    padding: 0 16px;
+    padding: 10px 16px;
     box-sizing: border-box;
+    font-size: 14px;
+    line-height: 1.4;
 
     display: flex;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)